### ISSUE TITLE
require dash core

### ISSUE THREAD

**Issue by alexcjohnson** [plotly/dash-component-boilerplate#103]
`dash` should be part of `install_requires` on the Python side, and `suggests` in R. Probably `>=1.0.0`?

**Comment by xhlulu** [comment]
Good idea! I've built dash-deck but didn't think about adding `dash` as a install_requires in `setup.py`. Now I realize that it might confuse newcomers if run into errors because they don't have `dash` installed before running `pip install dash-deck`.

As for the version, I wonder if it might be worth having the `dash-component-boilerplate` itself versioned? Once we have `dash` v2.0.0, do we still want the components created with the boilerplate to be compatible with all of v1.0.0, or would that likely not pose any issue?
